python/install.sh

```
@@ -124,18 +124,18 @@ get_latest_version() {
     echo "${latest_version}"
 }
 
-# Function to check if Python 3.11+ is available
+# Function to check if Python 3.12+ is available
 check_python() {
     if command -v python3 >/dev/null 2>&1; then
         local version=$(python3 -c "import sys; print(f'{sys.version_info.major}.{sys.version_info.minor}')")
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)